src/main/java/com/google/cloud/tools/app/impl/cloudsdk/CloudSdkAppEngineStandardStaging.java

```diff
@@ -82,7 +86,7 @@ public void stageStandard(StageStandardConfiguration configuration) throws AppEn
 
     try {
 
-      appEngineSdk.runCommand(arguments);
+      cloudSdk.runAppEngineSdkCommand(arguments);
```

Comment:
would potentially be interested in discussing the naming here runAppCfgCommand ?